src/components/shared/inputs/Pay/PayInputGroup.tsx

```diff
@@ -38,49 +38,58 @@ export default function PayInputGroup({
 
   const [payAmount, setPayAmount] = useState<string>('0')
   const [payInCurrency, setPayInCurrency] = useState<CurrencyOption>(ETH)
+  const [isErrorField, setIsErrorField] = useState<boolean>(false)
```

Comment:
**Suggestion:** If we continue with the state route (not ant d form validation), I think we can get a slightly better state name: maybe 'errorMessageVisible'